doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -102,9 +103,17 @@ Complete the following steps to configure the modem trace level at compile time:
 The application can use the :c:func:`nrf_modem_lib_trace_level_set` function to set the desired trace level.
 Passing ``NRF_MODEM_LIB_TRACE_LEVEL_OFF`` to the :c:func:`nrf_modem_lib_trace_level_set` function disables trace output.
 
+.. note::
+   The modem stores the current trace level on passing the ``AT+CFUN=0`` command.
+   If the trace level stored in the modem is ``NRF_MODEM_LIB_TRACE_LEVEL_OFF``, the application must enable traces in the modem using the :c:func:`nrf_modem_lib_trace_level_set` function or by enabling any of the Kconfig options mentioned in step 2.
```

Comment:
Is this the best way to refer to a step (step 2)? It just won't be updated if there are more steps.
@divipillai